tensorflow/tools/graph_transforms/graph_transforms_wrapper.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Wrapper for runninmg graph_transform binary embedded in pip site-package.
+NOTE: this mainly exists since PIP setup.py cannot install binaries to bin/.
+It can only install Python "console-scripts." This will work as a console
+script. See tools/pip_package/setup.py (search for CONSOLE_SCRIPTS).
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import sys
+import tensorflow as tf
+
+
+def main():
+  # Pip installs the binary in aux-bin off of main site-package install.
+  # Just find it and exec, passing all arguments in the process.
+  binary = os.path.join(tf.__path__[0], 'aux-bin/transform_graph')
```

Comment:
Can we make this work with both bazel, with having transform_graph as data dependency and pip environment?
We can have a condition in this script to check the file using resource_loader first, then use the aux_bin version?

Reply:
It should already be available through bazel: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/graph_transforms/README.md

Reply:
No, I mean making the wrapper work through bazel.
Or simply print an informational message if someone tries to run this through bazel.